test/configs/Util.ts

```diff
@@ -173,3 +175,20 @@ export const getBasicRequestParser = (bodyParsers: BodyParser[] = []): BasicRequ
  */
 export const getWebAclAuthorizer = (store: ResourceStore, aclManager = new UrlBasedAclManager()): WebAclAuthorizer =>
   new WebAclAuthorizer(aclManager, store);
+
+const mainModulePath = Path.join(__dirname, '../../');
+const loader = new Loader({ mainModulePath });
```

Comment:
We may want to consider re-creating the loader on each `instantiateFromConfig` call, as this loader holds module and config states.
Since we could be reusing config identifiers across different configs, but with different param values, these may confict, and cause unexpected instantiations.

This will have a small performance impact due to the repeated calls of `registerAvailableModuleResources`, but should be acceptable since these are integration tests.

Reply:
Will do!